### PR TITLE
[T4A5][W15-B4]Jasper Lee

### PR DESCRIPTION
**Pull request by jasperlee27**
//Assumption made as per instruction
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    public String getTag() {
+        String individualTag = "";
+
+        if (isAdded) {
+            individualTag = individualTag + "+ " + person.getName() + " " + tag.toString();
+            return individualTag;
+
+        } else {
+            individualTag = individualTag + "- " + person.getName() + " " + tag.toString();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:25]: can you not extract the common code outside of the `if ... else` block?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: header comments please!!

Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: header comments please!!

Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: header comments please!!

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+public class Tagging {
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: where are you maintaining the tagging list?

Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:6]: where are you maintaining the tagging list?

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 2 above)
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: header comments please!!

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 3 above)
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: where are you maintaining the tagging list?

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 2 above)
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: header comments please!!

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 3 above)
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: where are you maintaining the tagging list?

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 2 above)
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:4]: header comments please!

**Comment by okkhoy** [comment]
@jasperlee27 
Some comments added. Please ack & close the PR after reading comments.

**Comment by jasperlee27** [comment]
Oops thought tagging list is "assumed" under question.

**Pull request closed without merging**